.travis.yml

```diff
@@ -212,6 +225,18 @@ matrix:
         - DESC="build with OpenJDK11"
         - CMD="mvn -e package -Passembly && mvn -e site -Dlinkcheck.skip=true"
 
+    # OpenJDK12 build
+    - jdk: openjdk12
+      env:
+        - DESC="build with OpenJDK12"
+        - exclude1="\\!FileContentsTest#testGetJavadocBefore,\\!FileTextTest#testFindLine*,"
+        - exclude2="\\!MainFrameModelPowerTest#testOpenFileWithUnknownParseMode,"
+        - exclude3="\\!TokenUtilTest#testTokenValueIncorrect2,"
+        - exclude4="\\!ImportControlLoaderPowerTest#testInputStreamThatFailsOnClose"
+        - CMD1="mvn -e package -Passembly -Dtest=$exclude1$exclude2$exclude3$exclude4"
+        - CMD2="mvn -e site -Dlinkcheck.skip=true"
+        - CMD="$CMD1 && $CMD2"
```

Comment:
in travis we had one rule, it either sing line command and it stay in `travis.yml` or it goes to `travis.sh`.
It helps users(and me too) to quickly reuse single command, rather than figureout how to combine proper command from travis.
As we have `.ci` folder already ... we should use it to keep scripts for CI.

CIs execution commands should be single-line and short.

Reply:
Done.